### PR TITLE
feat(client): delete generic alert message 

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

The alert don't give a descriptive message, when campers interact with the save buttons.

Here is a better way to describe it, from Bruce comment here. https://github.com/freeCodeCamp/freeCodeCamp/pull/49216#discussion_r1103706841

> Although I do think it might help if we made those save alerts a little more specific. For example, if you save a change in the Personal Information, then instead of the alert just saying "We have updated your preferences" it might be nice if it said "We have saved your Personal Information changes".

---

Ref #49144.

### CODE REVIEW

**Comment by Sboonny** [comment]
currently it doesn't work, and I don't know why. 🤣

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I'll take a look at the tests next, but I found a few issues (mostly with the verbiage):

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -536,7 +543,8 @@
     "theme-updated": "Your theme has been updated!",
     "username-used": "Username is already associated with this account",
     "username-taken": "Username is already associated with a different account",
-    "username-updated": "We have updated your username to {{username}}",
+    "username-updated": "Your username is updated to {{username}}",
+    "privacy-updated": "Your privacy setting has been updated",
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/translations.json:547]:
```suggestion
    "username-updated": "We have updated your username to {{username}}",
    "privacy-updated": "We have updated your privacy settings",
```
This should fix the failing `username-change` tests.

File: api-server/src/server/boot/settings.js [hunk 2]
```diff
@@ -123,9 +133,14 @@ function updateMyPortfolio(...args) {
   const validate = ({ portfolio }) => portfolio?.every(isPortfolioElement);
   const isPortfolioElement = elem =>
     Object.values(elem).every(val => typeof val == 'string');
-  createUpdateUserProperties(buildUpdate, validate)(...args);
+  createUpdateUserProperties(
+    buildUpdate,
+    validate,
+    'flash.portoflio-item-updated'
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply api-server/src/server/boot/settings.js]: the test are working again, I think the issue was this typo

**Review by raisedadead — CHANGES_REQUESTED** [review]
This PR should be split into multiple parts because it introduces a breaking change. Please do the following:

- Introduce the new messages in the API side changes. 
- Wait for a deployment.
- Introduce the client-side changes using the new messages. 
- Wait for a deployment.
- Clean up the older messages, etc.

File: api-server/src/server/boot/settings.js [hunk 3]
```diff
@@ -148,9 +163,13 @@ function updateMyAbout(req, res, next) {
...
     ? { name, location, about, picture }
     : { name, location, about };
-  return user.updateAttributes(update, createStandardHandler(req, res, next));
+  return user.updateAttributes(
+    update,
+    createStandardHandler(req, res, next, 'flash.updated-about-me')
+  );
 }
 
+// why is this failing? I haven't touched it.
```
Comment (Sboonny) [inline api-server/src/server/boot/settings.js:172]:
```suggestion
```

**Comment by Sboonny** [comment]
Didn't mean to request review sorry about that

**Comment by ojeytonwilliams** [comment]
I think the order should be

1. expand the messages the client understands (including `updated-preferences`)
2. deploy
3. change the message the api sends
4. deploy
5. remove `updated-preferences` from client.

**Comment by Sboonny** [comment]
I am keeping the PR open, if someone wanna test it fully.

Feel free to close it, when the 3 PRs are merged

**Pull request closed without merging**